### PR TITLE
[docs] Fixed description of built-in audit rules of the module `runtime-audit-engine`

### PR DESCRIPTION
**Pull request by Suselz**
## Description
The documentation for the `runtime-audit-engine` module specified two sets of standard audit rules, without detailing what those rules were. This caused a problem of understanding for the untrained reader

## Why do we need it, and what problem does it solve?

The addition will help you better understand the module documentation

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: fix 
summary: Added description of built-in audit rules of the module `runtime-audit-engine`.
impact_level: low
```

### CODE REVIEW

File: ee/modules/650-runtime-audit-engine/docs/README_RU.md [hunk 1]
```diff
@@ -50,7 +50,8 @@ Deckhouse запускает агенты Falco (объединены в DaemonS
 Существуют два встроенных набора правил, которые нельзя отключить.
 Они помогают выявить проблемы с безопасностью Deckhouse и с самим модулем `runtime-audit-engine`:
 
-- `/etc/falco/k8s_audit_rules.yaml` — правила для аудита Kubernetes.
+- правило, статично размещенное в контейнере `falco`, по пути `/etc/falco/k8s_audit_rules.yaml` — правила для аудита Kubernetes.
+- правило, размещенное в формате custom resource [FalcoAuditRules](cr.html#falcoauditrules), `fstec` - правила аудита удовлетворяющие требованиям приказа ФСТЭК России №118 от 4 июля 2022г. (Требования по безопасности информации к средствам контейнеризации)
```
Comment (z9r5) [inline ee/modules/650-runtime-audit-engine/docs/README_RU.md:54]:
```suggestion
- правила, статично размещенные в контейнере `falco`, по пути `/etc/falco/k8s_audit_rules.yaml` — правила для аудита Kubernetes.
- правила, размещенные в формате custom resource [FalcoAuditRules](cr.html#falcoauditrules), `fstec` — правила аудита удовлетворяющие требованиям приказа ФСТЭК России №118 от 4 июля 2022г. (Требования по безопасности информации к средствам контейнеризации)
```

File: ee/modules/650-runtime-audit-engine/docs/README.md [hunk 2]
```diff
@@ -51,7 +51,8 @@ The main part of a rule is a conditional expression (which uses the [conditions
 
 There is a built-in set of rules that cannot be disabled. It helps to identify problems with Deckhouse security as well as security problems affecting the `runtime-audit-engine` module.
 
-- `/etc/falco/k8s_audit_rules.yaml` — Kubernetes audit rules.
+- a rule statically located in the `falco` container, along the path `/etc/falco/k8s_audit_rules.yaml` - rules for Kubernetes auditing.
+- a rule placed in the custom resource format [FalcoAuditRules](cr.html#falcoauditrules), `fstec` - audit rules that meet the requirements of Order No. 118 of the FSTEC of Russia dated July 4, 2022. (Information security requirements for containerization tools)
```
Comment (z9r5) [inline ee/modules/650-runtime-audit-engine/docs/README.md:55]:
```suggestion
- a rule statically located in the `falco` container, along the path `/etc/falco/k8s_audit_rules.yaml` — rules for Kubernetes auditing.
- a rule placed in the custom resource format [FalcoAuditRules](cr.html#falcoauditrules).
```

File: ee/modules/650-runtime-audit-engine/docs/README_RU.md [hunk 3]
```diff
@@ -50,7 +50,8 @@ Deckhouse запускает агенты Falco (объединены в DaemonS
 Существуют два встроенных набора правил, которые нельзя отключить.
 Они помогают выявить проблемы с безопасностью Deckhouse и с самим модулем `runtime-audit-engine`:
 
-- `/etc/falco/k8s_audit_rules.yaml` — правила для аудита Kubernetes.
+- правила, статично размещенные в контейнере `falco`, по пути `/etc/falco/k8s_audit_rules.yaml` — правила для аудита Kubernetes.
+- правила, размещенные в формате custom resource [FalcoAuditRules](cr.html#falcoauditrules), `fstec` — правила аудита удовлетворяющие требованиям приказа ФСТЭК России №118 от 4 июля 2022г. (Требования по безопасности информации к средствам контейнеризации)
```
Comment (z9r5) [inline ee/modules/650-runtime-audit-engine/docs/README_RU.md:54]:
```suggestion
- правила, размещенные в формате custom resource [FalcoAuditRules](cr.html#falcoauditrules), `fstec` — правила аудита удовлетворяющие требованиям приказа ФСТЭК России №118 от 4 июля 2022г. (Требования по безопасности информации к средствам контейнеризации).
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [9119](https://github.com/deckhouse/deckhouse/pull/9119) to the branch [release-1.62](https://github.com/deckhouse/deckhouse/tree/release-1.62) successful!

**Pull request merged**